packages/store-engine/package.json

```diff
@@ -2,11 +2,9 @@
   "dependencies": {
     "@types/bro-fs": "0.4.2",
     "@types/filesystem": "0.0.29",
-    "@types/fs-extra": "7.0.0",
     "@types/node": "~10",
```

Comment:
I think most of the `@types` dependencies can be moved to dev dependencies.

Reply:
Let's move the types on the next iteration, ok?